crates/uv/src/commands/project/lock.rs

```diff
@@ -40,6 +41,9 @@ pub(crate) async fn lock(
     // Discover or create the virtual environment.
     let venv = project::init(&project, cache, printer)?;
 
+    // Determine the set of installed packages.
+    let site_packages = SitePackages::from_executable(&venv)?;
```

Comment:
Should locking prefer installed packages? We don't do this in `pip compile`.

Reply:
Not sure...

Reply:
Removed.